File: Tareas/Ejercicio10.rb
=begin Nombre: Sergio Alberto Hernandez Serrato Mat: 1853132 Hora: V5*
Programa que imprime la serie A
=end
letrero = ""
puts("Que cantidad de numeros va a generar la serie?(Solo enteros)")
limite = gets().to_i
for i in (1..limite-1)
		operador = (2*i)-1
		resultado = i**operador
		letrero = letrero + resultado.to_s
		letrero = letrero + ", "
end
operador = (2*limite)-1
resultado = limite**operador
letrero = letrero + resultado.to_s
letrero = letrero + "."
if limite==0
	puts("No hay ningun numero")
else
	puts(letrero)
end

File: Tareas/Ejercicio10.java
/*Nombre: Sergio Alberto Hernandez Serrato Mat: 1853132 Hora: V5*
Programa que imprime la serie A*/


import javax.swing.JOptionPane;
public class Ejercicio10{
	public static void main(String[] args) {
		int  Limite, i, operador;
		String letrero="",Limitestring;
		double resultado;
		Limitestring = JOptionPane.showInputDialog("Cantidad de numeros a generar (solo enteros): ");
		Limite = Integer.parseInt(Limitestring);
			for (i=1; i<Limite; i++)
			{
				operador = (2*i)-1;
				resultado = Math.pow(i, operador);
				letrero= letrero+ resultado;
				letrero = letrero+ ", ";
			}
			operador = (2*Limite)-1;
				resultado = Math.pow(Limite, operador);
				letrero= letrero+ resultado;
				letrero = letrero+ ".";
		if(Limite==0)
			{
				JOptionPane.showMessageDialog(null, "No hay ningun numero");
			}
			else{
				JOptionPane.showMessageDialog(null, letrero);
			}
	}
}